packages/skill-lesson/utils/trpc-skill-lessons.ts

```diff
@@ -0,0 +1,34 @@
+import {httpBatchLink, loggerLink} from '@trpc/client'
+import {createTRPCNext} from '@trpc/next'
+import {inferRouterInputs, inferRouterOutputs} from '@trpc/server'
+import superjson from 'superjson'
+
+import {getBaseUrl} from './get-base-url'
+import {SkillLessonRouter} from '../server/routers/skill-lesson-router'
+
+export const trpcSkillLessons = createTRPCNext<SkillLessonRouter>({
+  config({ctx}) {
+    return {
+      transformer: superjson,
+      links: [
+        // adds pretty logs to your console in development and logs errors in production
+        loggerLink({
+          enabled: (opts) =>
+            process.env.NODE_ENV === 'development' ||
+            (opts.direction === 'down' && opts.result instanceof Error),
+        }),
+        httpBatchLink({
+          url: `${getBaseUrl()}/api/trpc`,
+        }),
+      ],
+      /**
+       * @link https://react-query.tanstack.com/reference/QueryClient
+       */
+      // queryClientConfig: { defaultOptions: { queries: { staleTime: 60 } } },
+    }
+  },
+  ssr: false,
+})
+
+export type RouterInput = inferRouterInputs<SkillLessonRouter>
+export type RouterOutput = inferRouterOutputs<SkillLessonRouter>
```

Comment:
this is another instance of the router, so technically we are calling this factory twice:

1. in the app using `AppRouter`
2. here in the `skill-lessons` package

This appears to work fine in practice. @KATT would be curious what you think 😅